runtime/bcverify/rtverify.c

```diff
@@ -526,6 +714,11 @@ verifyBytecodes (J9BytecodeVerificationData * verifyData)
 
 	/* walk the bytecodes linearly */
 	while (pc < length) {
+		if (isNPEMsg && pc >= npePC) {
```

Comment:
Why >= ? Should this not be == ?

Reply:
There are cases that the `pc` might be increased beyond `npePC` such as the case of `JBinvokeinterface2` right before the bytecode at `npePC`. https://github.com/eclipse/openj9/blob/9edff1b76ed55c0e40efb801dc83bc0a5b7c0a24/runtime/bcverify/rtverify.c#L623 increases current `pc` by `5` hence more than `npePC`.